raiden-dapp/src/components/TransferProgressDialog.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <v-card class="transfer-progress-dialog">
```

Comment:
I guess at some point we should create a `RaidenDialog` component that takes a component tree as children and displays it

Reply:
It would make sense yes :) I will try making something as part of the PR.

Reply:
Oh I didn't mean to extend this PR it was just in general :) i think we will have a ticket for this soon. But if you think it can be done quickly why not :)